postcss.config.js

```diff
@@ -1,5 +1,10 @@
 const autoprefixer = require('autoprefixer');
+const csso = require('postcss-csso')({restructure: true, comments: false});
```

Comment:
style should have spacing between curly braces, e.g.
`{restructure: true, comments: false}`
to
`{ restructure: true, comments: false }`